nixos/modules/services/security/vaultwarden/default.nix

```
@@ -65,6 +65,8 @@ let
 
   vaultwarden = cfg.package.override { inherit (cfg) dbBackend; };
 
+  useSendmail =
+    (configEnv ? SENDMAIL_COMMAND) && (configEnv ? USE_SENDMAIL) && (configEnv.USE_SENDMAIL == "true");
```

`SENDMAIL_COMMAND` doesn't need to be specified if you add it to `systemd.services.vaultwarden.path`.
```suggestion
  useSendmail = configEnv.USE_SENDMAIL or null == "true";
```